storage/constants.go

```diff
@@ -0,0 +1,35 @@
+package storage
+
+import (
+	"time"
+)
+
+// MaxRetriesToCreateDB represents the maximum number of times to try to create DB if it failed
+const MaxRetriesToCreateDB = 10
```

Comment:
as this is used only in the `storage` package and it also exists in `elrond-go-storage`, why not use the one from `elrond-go-storage` repo? this way we avoid missing updating one of them in one of the repos

same for `SleepTimeBetweenCreateDBRetries`

Reply:
the constants from elrond-go-storage were removed:
https://github.com/ElrondNetwork/elrond-go-storage/pull/4/files